.cicd/create-docker-from-binary.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# buildkite-agent artifact download someversion.deb .
+
+buildkite-agent artifact download '*.deb' --step ':ubuntu: Ubuntu 18.04 - Package Builder' .
+echo ":done: download successfull"
+
+#PREFIX='base-ubuntu-18.04'
```

Comment:
Please remove commented-out code. Our stance on commented-out code in base branches is that we can retrieve old code from version control, if we need it.

Reply:
Done!